src/routes/Profile/EducationHistory/EducationHistoryWizard/EducationHistoryWizard.tsx

```diff
@@ -61,11 +61,12 @@ const EducationHistoryWizard = ({ isOpen, onClose }: EducationHistoryWizardProps
               osaamiset: z.array(
                 z.object({
                   id: z.string().min(1),
+                  nimi: z.object({}).catchall(z.string()),
+                  kuvaus: z.object({}).catchall(z.string()),
                 }),
               ),
             })
-            .array()
-            .nonempty(),
```

Comment:
I thought this nonempty to be not required as there is that `.refine` where it is checking the same thing.